src/app/tasks/tasks.service.ts

```diff
@@ -85,4 +89,15 @@ export class TasksService {
     );
   }
 
+  removeAssigneeFromTask(assignee: any = '') {
+    const removeAssignee = assignee._id === assignee.userId;
+    const link = `/teams/view/${this.link.teams}`;
+
+    if (removeAssignee) {
+      return this.couchService.findAll(this.dbName, findDocuments(this.getTasks())).pipe(
+        switchMap((docs: any[]) => this.couchService.bulkDocs(this.dbName, docs.map(doc => ({ ...doc, _deleted: true }))))
```

Comment:
This will delete the task, but we only want to remove the assignee.

`({ ...doc, assignee: '' })`